experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -136,6 +158,37 @@ export class FieldProxyTarget extends ProxyTarget<FieldAnchor> implements Editab
 		return this.cursor.getFieldLength();
 	}
 
+	/* eslint-disable no-param-reassign */
+
+	public slice(start?: number, end?: number) {
+		const length = this.length;
+
+		start = normalizeIndex(start, length);
+		if (start < 0) start = 0;
```

Comment:
This just handles the negative infinity case? I think it would be better to error than silently convert it to 0.

Reply:
It also handles the start/end being < -array.length, which gets this special behavior.